### PR TITLE

Adding functionality for specifying languages as parameters in the CLI.


### PR DESCRIPTION
**Pull request by MartinSandstrom**
Ignoring test-gitignore file

Adding parameter input

Adding first functionality for getting the data.

### CODE REVIEW

**Comment by seriema** [comment]
Update the readme to include some instructions on how to use this?

**Comment by MartinSandstrom** [comment]
Readme is updated with instructions.

File: readme.md [hunk 1]
```diff
@@ -1,64 +1,33 @@
...
+#### params
+
+Type: `array`
+
+Example: java node c
 
 ## License
 
-MIT © [<%= name %>](https://github.com/kvasir/gitignore-cli)
+MIT © gitignore-cli (https://github.com/kvasir/gitignore-cli)
```
Comment (seriema) [inline readme.md]: The brackets still need to be there to make the link clickable. Check the "view" options to see it rendered in Github.

**Comment by MartinSandstrom** [comment]
Brackets re added.

**Pull request merged**